src/range_proof.rs

```diff
@@ -56,13 +59,13 @@ mod tests {
             let (min, max) = (0u64, ((1u128 << n) - 1) as u64);
             let values: Vec<u64> = (0..m).map(|_| rng.gen_range(min, max)).collect();
             for v in values {
-                assert!(range_proof_helper(v, *n).is_ok());
+                assert!(range_proof_helper(SignedInteger::Positive(v), *n).is_ok());
             }
-            assert!(range_proof_helper(max + 1, *n).is_err());
+            assert!(range_proof_helper(SignedInteger::Positive(max + 1), *n).is_err());
         }
     }
 
-    fn range_proof_helper(v_val: u64, n: usize) -> Result<(), R1CSError> {
+    fn range_proof_helper(v_val: SignedInteger, n: usize) -> Result<(), R1CSError> {
```

Comment:
i think we can keep u64 because we want to permit rangeproofs only on positive ints.